go/internal/demo/helper.go

```diff
@@ -1,6 +1,7 @@
 package demo
 
 import (
+	"github.com/gin-gonic/gin"
```

Comment:
[nit] I prefer external dependencies to be below stdlib imports (basically files to be `goimports` formatted)

```
import (
	"net/http"
	"os"

	"github.com/gin-gonic/gin"
)
```

Reply:
TBH, I tend not to look at the imports section (it's usually collapsed in my IDE). BUT I've updated my default IDE settings to use ``goimports` style for imports.... not more nits 🪳